src/components/HelpTopicLink/HelpTopicLink.test.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { render, cleanup, screen, fireEvent } from '@testing-library/react';
+import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
+import { HelpTopicLink } from './';
+
+jest.mock('@redhat-cloud-services/frontend-components/useChrome', () => ({
+  useChrome: jest.fn(),
+}));
+
+const useChromeMock = useChrome as jest.Mock;
+
+describe('HelpTopicLink', () => {
+  test('should enable topic when component is mounted and disable when unmounted', () => {
+    const enableTopics = jest.fn();
+    const disableTopics = jest.fn();
+    useChromeMock.mockReturnValue({
+      helpTopics: { setActiveTopic: jest.fn(), enableTopics, disableTopics },
+    });
+
+    render(<HelpTopicLink topicId="test" />);
+    expect(enableTopics).toHaveBeenCalledWith('test');
+    cleanup();
+    expect(disableTopics).toHaveBeenCalledWith('test');
+  });
+
+  test('should set active topic when button is clicked', () => {
+    const setActiveTopic = jest.fn();
+    useChromeMock.mockReturnValue({
+      helpTopics: { setActiveTopic, enableTopics: jest.fn(), disableTopics: jest.fn() },
+    });
+
+    render(<HelpTopicLink topicId="test">Learn more</HelpTopicLink>);
+    fireEvent.click(screen.getByText('Learn more'));
```

Comment:
I mentioned this to @kdoberst, but I tend to favor using `data-` tags over text as this could break with translations, or if `Learn more` appears somewhere else earlier in the DOM. Won't block this PR on it as we need a broader discussion on the matter. Perhaps in arch sync.